task/03-date-tasks.js

```diff
@@ -74,8 +75,9 @@ function isLeapYear(date) {
  *    Date(2000,1,1,10,0,0),  Date(2000,1,1,10,0,0,250)     => "00:00:00.250"
  *    Date(2000,1,1,10,0,0),  Date(2000,1,1,15,20,10,453)   => "05:20:10.453"
  */
-function timeSpanToString(startDate, endDate) {
-  throw new Error('Not implemented');
+function timeSpanToString(startDate, endDate) { 
+  const duration = endDate.getTime() - startDate.getTime();
+  return new Date(duration).toISOString().slice(11, 23);  
```

Comment:
When you introduce some values, try to give meaningful variables for numbers or strings. Try to avoid magic values

Reply:
renamed a variable